lib/rubocop/cop/rspec/verified_doubles.rb

```diff
@@ -9,13 +9,18 @@ module RSpec
       #
       # @example
       #   # bad
-      #   it '...' do
-      #     widget = double("Widget")
+      #   let(:foo) do
+      #     widget = double(method_name: 'returned value')
```

Comment:
The assignment (`widget =`) can be removed.

Reply:
Oops, I forget it. I'll remove them soon!